### ISSUE TITLE
Prelaunch fails with argument +append

### ISSUE THREAD

**Issue by camall3n** [camall3n/onager#25]
Argument must be `+a` or `++append`.

Suggest changing to `+append`.

**Comment by camall3n** [comment]
Not fixed.

```
onager prelaunch +arg --foo 1 2 3 +arg --bar a b c +command "./examples/myscript" +jobname test
Traceback (most recent call last):
  File "/Users/cam/dev/onager/venv/bin/onager", line 7, in <module>
    exec(compile(f.read(), __file__, 'exec'))
  File "/Users/cam/dev/onager/bin/onager", line 14, in <module>
    meta_launcher.meta_launch(args)
  File "/Users/cam/dev/onager/onager/meta_launcher.py", line 97, in meta_launch
    if args.append:
AttributeError: 'Namespace' object has no attribute 'append'
```

**Comment by camall3n** [comment]
```
onager prelaunch +arg --foo 1 2 3 +arg --bar a b c +command "./examples/myscript" +jobname test +append
Traceback (most recent call last):
  File "/Users/cam/dev/onager/venv/bin/onager", line 7, in <module>
    exec(compile(f.read(), __file__, 'exec'))
  File "/Users/cam/dev/onager/bin/onager", line 14, in <module>
    meta_launcher.meta_launch(args)
  File "/Users/cam/dev/onager/onager/meta_launcher.py", line 97, in meta_launch
    if args.append:
AttributeError: 'Namespace' object has no attribute 'append'
```

**Comment by camall3n** [comment]
I'm working on a fix for #26 as well that should fix this too
